kedro-datasets/kedro_datasets_experimental/opik/README.md

```diff
@@ -0,0 +1,451 @@
+    },
+)
+
+# Returns Opik Prompt object
+prompt_obj = dataset.load()
+
+# Access prompt content
+content = prompt_obj.prompt
+
+# Access metadata
+metadata = prompt_obj.metadata
+```
+
+##### LangChain Mode
+Returns ready-to-use `ChatPromptTemplate` objects:
+```python
+dataset = OpikPromptDataset(mode="langchain", ...)
+
+# ChatPromptTemplate object
+template = dataset.load()
+
+# Ready to use with LangChain
+formatted = template.format(question="What is Kedro?")
+```
+
+### Configuration Examples
+
+#### Catalog Configuration (YAML)
+
+##### Local Sync Policy - Development
+
+```yaml
+dev_prompt:
+  type: kedro_datasets_experimental.opik.OpikPromptDataset
+  filepath: data/prompts/customer_support.json
+  prompt_name: customer_support_v1
+  prompt_type: chat
+  credentials: opik_credentials
+  sync_policy: local  # Local files are source of truth
+  mode: langchain
+  save_args:
+    metadata:
+      environment: development
+      version: "2.1"
+```
+
+##### Remote Sync Policy - Production
+```yaml
+production_prompt:
+  type: kedro_datasets_experimental.opik.OpikPromptDataset
+  filepath: data/prompts/production.json
+  prompt_name: customer_support_v1
+  prompt_type: chat
+  credentials: opik_credentials
+  sync_policy: remote  # Opik is source of truth
+  mode: sdk
+```
+
+##### Strict Sync Policy - CI/CD
+```yaml
+validation_prompt:
+  type: kedro_datasets_experimental.opik.OpikPromptDataset
+  filepath: data/prompts/validation.yaml
+  prompt_name: customer_support_v1
+  prompt_type: chat
+  credentials: opik_credentials
+  sync_policy: strict  # Error if local and remote differ
+  mode: sdk
+```
+
+#### Python API Examples
+
+##### Basic Usage
+```python
+from kedro_datasets_experimental.opik import OpikPromptDataset
+
+# Minimal configuration
+dataset = OpikPromptDataset(
+    filepath="prompts/support.json",
+    prompt_name="support_assistant",
+    credentials={
+        "api_key": "opik_...",  # pragma: allowlist secret
+        "workspace": "my-workspace",
+    },
+)
+```
+
+##### Advanced Configuration
+```python
+# Full configuration with metadata
+dataset = OpikPromptDataset(
+    filepath="prompts/assistant.yaml",
+    prompt_name="assistant_v2",
+    prompt_type="chat",
+    sync_policy="remote",
+    mode="langchain",
+    credentials={
+        "api_key": "opik_...",  # pragma: allowlist secret
+        "workspace": "my-workspace",
+        "project_name": "customer-support",
+    },
+    save_args={
+        "metadata": {
+            "environment": "production",
+            "team": "ml-ops",
+            "version": "2.0.0"
+        }
+    },
+)
+```
+
+#### Credentials Management
+
+##### Configuration
+```yaml
+# conf/local/credentials.yml
+# Store securely - not in version control
+opik_credentials:
+  api_key: "opik_your_api_key"  # pragma: allowlist secret
+  workspace: "your-workspace"
+  project_name: "your-project"  # optional
+```
+
+##### Shared Credentials with OpikTraceDataset
```

Comment:
This still doesn't work from me, and from @ElenaKhaustova 's comment I figured it's also not supposed to for now?